.rspec

```diff
@@ -1,4 +0,0 @@
---color
---format documentation
---backtrace
---default_path spec
```

Comment:
Because rspec wouldn't run with any of those options. I assume they were removed.